sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1313,131 +1313,165 @@ object functions {
+   * @param xName coordinate on x-axis
+   * @return the <i>theta</i> component of the point
+   *         (<i>r</i>,&nbsp;<i>theta</i>)
+   *         in polar coordinates that corresponds to the point
+   *         (<i>x</i>,&nbsp;<i>y</i>) in Cartesian coordinates,
+   *         as if computed by [[java.lang.Math#atan2]]
    *
    * @group math_funcs
    * @since 1.4.0
    */
-  def atan2(leftName: String, rightName: String): Column =
-    atan2(Column(leftName), Column(rightName))
+  def atan2(yName: String, xName: String): Column =
+    atan2(Column(yName), Column(xName))
 
   /**
-   * Returns the angle theta from the conversion of rectangular coordinates (x, y) to
-   * polar coordinates (r, theta).
+   * @param y coordinate on y-axis
+   * @param xValue coordinate on x-axis
+   * @return the <i>theta</i> component of the point
+   *         (<i>r</i>,&nbsp;<i>theta</i>)
+   *         in polar coordinates that corresponds to the point
+   *         (<i>x</i>,&nbsp;<i>y</i>) in Cartesian coordinates,
+   *         as if computed by [[java.lang.Math#atan2]]
    *
    * @group math_funcs
    * @since 1.4.0
    */
-  def atan2(l: Column, r: Double): Column = atan2(l, lit(r))
+  def atan2(y: Column, xValue: Double): Column = atan2(y, lit(xValue))
 
   /**
-   * Returns the angle theta from the conversion of rectangular coordinates (x, y) to
-   * polar coordinates (r, theta).
+   * @param yName coordinate on y-axis
+   * @param xValue coordinate on x-axis
+   * @return the <i>theta</i> component of the point
+   *         (<i>r</i>,&nbsp;<i>theta</i>)
+   *         in polar coordinates that corresponds to the point
+   *         (<i>x</i>,&nbsp;<i>y</i>) in Cartesian coordinates,
+   *         as if computed by [[java.lang.Math#atan2]]
    *
    * @group math_funcs
    * @since 1.4.0
    */
-  def atan2(leftName: String, r: Double): Column = atan2(Column(leftName), r)
+  def atan2(yName: String, xValue: Double): Column = atan2(Column(yName), xValue)
 
   /**
-   * Returns the angle theta from the conversion of rectangular coordinates (x, y) to
-   * polar coordinates (r, theta).
+   * @param yValue coordinate on y-axis
+   * @param x coordinate on x-axis
+   * @return the <i>theta</i> component of the point
+   *         (<i>r</i>,&nbsp;<i>theta</i>)
+   *         in polar coordinates that corresponds to the point
+   *         (<i>x</i>,&nbsp;<i>y</i>) in Cartesian coordinates,
+   *         as if computed by [[java.lang.Math#atan2]]
    *
    * @group math_funcs
    * @since 1.4.0
    */
-  def atan2(l: Double, r: Column): Column = atan2(lit(l), r)
+  def atan2(yValue: Double, x: Column): Column = atan2(lit(yValue), x)
 
   /**
-   * Returns the angle theta from the conversion of rectangular coordinates (x, y) to
-   * polar coordinates (r, theta).
+   * @param yValue coordinate on y-axis
+   * @param xName coordinate on x-axis
+   * @return the <i>theta</i> component of the point
+   *         (<i>r</i>,&nbsp;<i>theta</i>)
```

Comment:
nit `&nbsp;` -> ` `. Seems broken in Javadoc.

Reply:
Ok. I have removed the non breaking spaces.